contrib_bots/lib/githubcomment.py

```diff
@@ -0,0 +1,130 @@
+# See readme.md for instructions on running this code.
+from github import GithubConn
+import requests
+import json
+
+
+class GitHubHandler(object):
+    '''
+    This plugin facilitates allows you to comment on a GitHub issue, under a certain repository.  It
+    looks for messages starting with '@comment' or '@github-comment.
+    '''
+
+    def usage(self):
+        return '''
+            This bot will allow users to comment on a GitHub issue.  Users should preface
+            messages with "@comment" or "@github-comment". You will need to have a GitHub account.
+
+            Before running this, make sure to get a GitHub OAuth token, and store it in the github_token.txt file.
+            Also, edit the path_to_token variable in the send_to_github function to be the path to your token.
+            Please input info like this: /<username>/<repository_owner>/<repository>/<issue_number>/<comment>
+            '''
```

Comment:
The path should be in quotation marks, there is a missing full stop at the end.

Why is the `<comment>` necessary at the end of the path?